plutus-contract/src/Wallet/Emulator/Folds.hs

```diff
@@ -131,7 +131,12 @@ instanceState con tag =
             case flt e of
                 Nothing -> pure Nothing
                 Just response -> case traverse (JSON.fromJSON @(Event s)) response of
-                    JSON.Error e'   -> throwError $ JSONDecodingError e' response
+                    JSON.Error e'   ->
```

Comment:
When I was trying to track this bug down, it took me a while to find that this was the spot where the error was being thrown (there's a few places with very similar code); maybe we could have a `logError` (or similar) with a hint, such as the one in this comment, here as well?

Reply:
Good point. I'll add a `logError` message.